lib/classes/Variables.js

```diff
@@ -97,14 +97,14 @@ class Variables {
     let finalValue;
     const variableStringsArray = variableStringsString.split(',');
     variableStringsArray.find(variableString => {
-      finalValue = this.getValueFromSource(variableString);
+      finalValue = this.getValueFromSource(variableString, true);
       return (typeof finalValue !== 'undefined' && finalValue !== null);
     });
 
     return finalValue;
   }
 
-  getValueFromSource(variableString) {
+  getValueFromSource(variableString, overwriteMode) {
```

Comment:
imo I don't think this overwrite parameter is necessary ... any specific reason why you have it? it seems that it'll always be true.